dashboard/settings.py

```diff
@@ -282,7 +288,23 @@
         'handlers': ['console']
     },
 }
-
+RANDOM_PASSWORD_DEFAULT_LENGTH = ENV.get('RANDOM_PASSWORD_DEFAULT_LENGTH', 32)
+DB_CACHE_CONFIGS = ENV.get('DB_CACHE_CONFIGS',
+                           {'CACHE_TTL': 7200, 'BACKEND': import_string('django_mysql.cache.MySQLCache'),
+                            'LOCATION': 'django_myla_cache',
+                            'CACHE_KEY_PREFIX': 'myla',
+                            'CACHE_OPTIONS': {'COMPRESS_MIN_LENGTH': 5000, 'COMPRESS_LEVEL': 6}
+                            })
+
+CACHES = {
```

Comment:
It doesn't look like there's any easy way to disable this cache. I'm not sure if that's something we'll need to do.

When I setup Student Explorer [I had DummyCache as the default](https://github.com/tl-its-umich-edu/student_explorer/blob/2d5445a7b5ec0f7da8e634d7676c67a464ca4404/student_explorer/settings.py#L387). This might work better, but then someone might not configure it. 

This is probably fine though as a default.